src/components/accessories/appHeader/AppHeader.tsx

```diff
@@ -113,14 +113,14 @@ const AppHeader: FunctionComponent<TProps> = ({
           </div>
           <div className="appHeader__nav">
             <div className="appHeader__nav_items">
-              <div className="appHeader__nav__item">{t("nav.pharmacy")}</div>
-              <div className="appHeader__nav__item">{t("nav.ward")}</div>
               <div
                 className="appHeader__nav__item"
-                onClick={() => history.push("/billing")}
+                onClick={() => history.push("/patients")}
```

Comment:
There is no "/patients" route for now, please just change this in "/" route.

Reply:
@axelpezzo @SteveGT96 Actually, the new routing dispositions are something that we might handle in the [OP-827](https://openhospital.atlassian.net/browse/OP-827). If beyond it there's no other question, perhaps we can resolve these conversations and consider this PR approved. What do you guys think?

Reply:
I think that's OK!